examples/dfcx_agent_cicd/cicd_code/readme.md

```diff
@@ -0,0 +1,331 @@
+    to find impacted flow the current instance of CI/CD builds.
+
+## To Set up the pipeline 
+
+1.  Setup a git or any code repository of your choice to store the code
+    and agent artifacts
+
+2.  Push the code you see in the parent folder along with this
+    documentation.
+
+3.  Make sure to update the command in the code base in
+    export/repopush.sh file in line
+    #7 to point to your git you have created in step 1 as this would
+    check in the agent artifacts back in the same repo as your code is
+    in a folder called Agent Artifacts.
+
+4.  Use the config file as a one stop place to initiate values to
+    variables that will be used throughout the pipeline. Hence this
+    eases out the maintenance or reusing of the pipeline for different
+    values
+
+{
+
+\"agent_name\" : \"carrental\",
+
+\"dev_env_pull\" : \"ready to deploy\",
+
+\"uat_env_deploy\" : \"ready to test\",
+
+\"prod_env_deploy\" :\"deployed\",
+
+\"devprodsyncenv\" :\"deployed\",
+
+\"bucket\": \"DFCX_agent_cicd_export\",
+
+\"dev_project\": \"yourprojectid\",
+
+\"uat_project\" : \"yourprojectid\",
+
+\"prod_project\": \"yourprojectid\",
+
+\"devops_project\": \"yourprojectid\",
+
+\"uat_webhook_env\": \"uat\",
+
+\"prod_webhook_env\": \"prod\",
+
+\"uat_engine_id\" :\"\",
+
+\"prod_engine_id\" :\"\"
+
+}
+
+
+
+5.  Make sure the a GCP bucket is created with said structure and name
+    is configured in config file
+
+6.  Create 3 cloud builds with the configuration and name as shown in
+    screenshots in the previous section and attach your repo to these
+    builds.
+
+7.  Make sure an agent is present in the same name in UAT and Prod(if it
+    is first time, just create an empty agent in UAT/Prod projects)
+
+8.  Make sure the agent in UAT and Prod projects has the environments
+    created as configured in config file in fields uat_env_deploy and
+    prod_env_deploy
+
+9.  Make sure you have also created the env as you configured in config
+    file devprodsyncenv in all UAT and Dev projects to sync back the
+    flows after deployed in prod
+
+## To run the Pipeline
+
+1.  Now make some changes to your agent in Dev project and create a
+    version of the flow in dfcx and deploy updated flows to the DFCX
+    environment as you have configured in the config file dev_env_pull
+    field.
+
+2.  Now come to GCP Cloud build console and click on RUN on exportbuild
+    in triggers section and input the commit message(basically some
+    lines about your change in the agent that will be used )
+
+3.  This will export agent and this would have done a check in in the
+    repo to trigger UAT and prod builds and deployed the agent in UAT
+    project.
+
+4.  Now you can come back to cloud build console and build history tab
+    and approve the build that is waiting for your approval and you can
+    see that it will deploy the agent in prod post approval
+    !media/image6.png
```

Comment:
this image is not loading in the markdown file

Reply:
Fixed in my latest push to my branch